dev/NavigationView/NavigationView.vcxitems

```diff
@@ -9,6 +9,7 @@
   <ItemDefinitionGroup>
     <ClCompile>
       <AdditionalIncludeDirectories>%(AdditionalIncludeDirectories);$(MSBuildThisFileDirectory)</AdditionalIncludeDirectories>
+      <PreprocessorDefinitions>%(PreprocessorDefinitions);NAVIGATIONVIEW_INCLUDED</PreprocessorDefinitions>
```

Comment:
Why can these definitions not go in the common FeatureAreas.props?

Reply:
I like that these are packaged along with the projects and not outside of them. This can be done for the mux dll since all of them build into one. For the tests I had to create them in the props file but I'm getting rid of them in the props file altogether and using reflection instead.

---
In reply to: [302730563](https://github.com/microsoft/microsoft-ui-xaml/pull/956#discussion_r302730563) [](ancestors = 302730563)